### ISSUE TITLE
Show list elements after the $ sign without pressing an extra tab

### ISSUE THREAD

**Issue by chisqr** [randy3k/radian#163]
The autocomplete drop down is working fine when getting function names etc. But when I try to get the column names from a list like x$col_name, I have to press a tab after x$. Will it be possible to show the drop down immediately after the $ from the context of the statement ?

**Comment by randy3k** [comment]
Currently, the auto-completion is triggered if you entered two alphanumerics in a row. If you set `options(radian.completion_prefix_length = 0)` in `.Rprofile`. It will trigger completion for everything that you entered.

**Comment by chisqr** [comment]
Yes I saw that, and I like it that way. I was thinking of giving characters like $, @ special treatment.
